[2024-12-17T10:18:37.547+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T10:18:37.595+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task scheduled__2024-12-16T00:29:00+00:00 [queued]>
[2024-12-17T10:18:37.618+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v3.third_task scheduled__2024-12-16T00:29:00+00:00 [queued]>
[2024-12-17T10:18:37.621+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T10:18:37.659+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): third_task> on 2024-12-16 00:29:00+00:00
[2024-12-17T10:18:37.683+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1615) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T10:18:37.693+0000] {standard_task_runner.py:72} INFO - Started process 1641 to run task
[2024-12-17T10:18:37.691+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v3', 'third_task', 'scheduled__2024-12-16T00:29:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpl04id6ns']
[2024-12-17T10:18:37.702+0000] {standard_task_runner.py:105} INFO - Job 122: Subtask third_task
[2024-12-17T10:18:37.852+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v3.third_task scheduled__2024-12-16T00:29:00+00:00 [running]> on host 8348eb091483
[2024-12-17T10:18:38.138+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='srush' AIRFLOW_CTX_DAG_ID='my_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T00:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-16T00:29:00+00:00'
[2024-12-17T10:18:38.140+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T10:18:38.178+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-17T10:18:38.181+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task3 runs after task1 and with task2']
[2024-12-17T10:18:38.221+0000] {subprocess.py:99} INFO - Output:
[2024-12-17T10:18:38.227+0000] {subprocess.py:106} INFO - task3 runs after task1 and with task2
[2024-12-17T10:18:38.253+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-17T10:18:38.431+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T10:18:38.436+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v3, task_id=third_task, run_id=scheduled__2024-12-16T00:29:00+00:00, execution_date=20241216T002900, start_date=20241217T101837, end_date=20241217T101838
[2024-12-17T10:18:38.548+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T10:18:38.595+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-17T10:18:38.602+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
